generators/Generators.fs

```diff
@@ -1496,9 +1496,27 @@ type Wordy() =
 type Yacht() =
     inherit GeneratorExercise()
 
+    let dieValueToEnumName = function
+        | 1 -> "One"
+        | 2 -> "Two"
+        | 3 -> "Three"
+        | 4 -> "Four"
+        | 5 -> "Five"
+        | 6 -> "Six"
+        | n -> failwith ("Invalid die value: " + n.ToString())
+
+    let formatDieList (value: JToken) =
```

Comment:
You might be able to simplify this a bit. For inspiration, consider the code in the [allergies generator](https://github.com/exercism/fsharp/blob/master/generators/Generators.fs#L49), which does a similar thing (rendering a list of discriminated unions).